### PR TITLE
Feat/KFS-1175 display current cfu

### PR DESCRIPTION
**Pull request by Guttz**
Release Notes
-------------

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----

We now display the current amount of cfus being used by a compute pool.

References
----------

Test & Review
-------------

snapshot tests/golden files

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -125,8 +125,12 @@ github.com/confluentinc/ccloud-sdk-go-v2/connect v0.4.0 h1:sv7Ss7WqKkVAXmXVlR3VF
 github.com/confluentinc/ccloud-sdk-go-v2/connect v0.4.0/go.mod h1:vzL/maQvg76G8BMXM9Mx0FNrXI1ctdBr46/EbZ0VJso=
 github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin v0.0.1 h1:cEp0C29HMCq0eR9Zv8RGARUGJjjElpzNUmQp66qKHkA=
 github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin v0.0.1/go.mod h1:AaF39Acy3LFnHSHExaUtqNmbs7kL5/AL54CXX61+Il8=
-github.com/confluentinc/ccloud-sdk-go-v2/flink v0.4.0 h1:ueXax5e70pLN6yQ6vSJVBlAAGrb0Js+tJO7kYzxoAU8=
-github.com/confluentinc/ccloud-sdk-go-v2/flink v0.4.0/go.mod h1:x+8kpYsJHRlvGuIB/tV0afPNyjKst3MsNOE6XsjgAl0=
+github.com/confluentinc/ccloud-sdk-go-v2/flink v0.5.0 h1:5lh7TY2aUlZA0wUL0wAebZeGPp8uEnEhZawjYhnRUxo=
+github.com/confluentinc/ccloud-sdk-go-v2/flink v0.5.0/go.mod h1:x+8kpYsJHRlvGuIB/tV0afPNyjKst3MsNOE6XsjgAl0=
```
Comment (brianstrauch) [inline go.sum:129]: `go mod tidy`

File: internal/flink/command_compute_pool.go [hunk 2]
```diff
@@ -12,12 +12,13 @@ import (
...
-	IsCurrent bool   `human:"Current" serialized:"is_current"`
-	Id        string `human:"ID" serialized:"id"`
-	Name      string `human:"Name" serialized:"name"`
-	Cfu       int32  `human:"CFU" serialized:"cfu"`
-	Region    string `human:"Region" serialized:"region"`
-	Status    string `human:"Status" serialized:"status"`
+	IsCurrent  bool   `human:"Current" serialized:"is_current"`
+	Id         string `human:"ID" serialized:"id"`
+	Name       string `human:"Name" serialized:"name"`
+	CurrentCfu int32  `human:"Current CFU" serialized:"currrent_cfu"`
```
Comment (brianstrauch) [inline internal/flink/command_compute_pool.go:18]: This is a breaking change, so please update https://confluentinc.atlassian.net/wiki/spaces/DPI/pages/2907775380/Feature+Flag+Breaking+Changes

Comment (Guttz) [reply internal/flink/command_compute_pool.go:18]: updated

**Review by brianstrauch — APPROVED** [review]
Thanks!!

**Pull request merged**